include/bbp/sonata/common.h

```diff
@@ -39,6 +40,7 @@ SONATA_API const std::string version();
 using NodeID = uint64_t;
 using EdgeID = uint64_t;
 using ElementID = uint32_t;
+using CompartmentID = std::array<uint64_t, 2>;
```

Comment:
can we add a comment here saying that this should be pairs of `<typeof(NodeID), typeof(ElementID)>`, but that we're using the extra space such that the bindings are (much!) faster?

Reply:
Done!